java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S1313_java.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "IP addresses should not be hardcoded",
-  "type": "VULNERABILITY",
+  "type": "SECURITY_HOTSPOT",
```

Comment:
is it ok that there is no security standards for it?

Reply:
I don't know. @agigleux did we miss some standards for RSPEC-1313?

Reply:
Fixed